### PR TITLE
Migrate more of the storage package from client.KV to client.DB.

### PR DESCRIPTION
**Pull request by petermattis**
See #997.

### CODE REVIEW

File: client/db.go [hunk 1]
```diff
@@ -438,6 +438,19 @@ func (tx *Tx) Get(key interface{}) (KeyValue, error) {
 	return runOneRow(tx, tx.B.Get(key))
 }
 
+// GetProto retrieves the value for a key and decodes the result as a proto
+// message.
+//
+// key can be either a byte slice, a string, a fmt.Stringer or an
+// encoding.BinaryMarshaler.
+func (tx *Tx) GetProto(key interface{}, msg gogoproto.Message) error {
```
Comment (bdarnell) [inline client/db.go]: We should add this to Batch and batcher too. (and maybe add a test that uses reflection to compare the methods on each type)

Comment (petermattis) [inline client/db.go]: Would be good to compare the methods on `DB` and `Tx` using reflection. Somewhat harder to add this to `Batch` and `Batcher` as those gets are not synchronous. Suppose I can use the `Call.Post` mechanism. I'll do that in a separate change.

**Comment by petermattis** [comment]
Heh, clicked merge on the wrong PR. If there are any comments I'll address in a subsequent PR.

**Comment by bdarnell** [comment]
LGTM

**Pull request merged**